googleapiclient/http.py

```diff
@@ -81,23 +84,76 @@ def _retry_request(http, num_retries, req_type, sleep, rand, uri, method, *args,
   Returns:
     resp, content - Response from the http request (may be HTTP 5xx).
   """
+
```

Comment:
Drop this inserted blank line. Unless there's some reason for it that I'm missing?